sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -184,35 +178,38 @@ DocumentAnalysisClient documentAnalysisClient = new DocumentAnalysisClientBuilde
-    - Receipts
-    - Business cards
-    - Invoices
-    - Identity Documents
-    - Documents
-    
-  See example [Prebuilt Models](#use-prebuilt-models).
+provide both synchronous and asynchronous operations for analyzing input documents using custom and prebuilt models
+through the `beginAnalyzeDocument` and `beginAnalyzeDocumentFromUrl` methods.
+Use the `model` parameter to select the type of model for analysis.
+
+|Model| Features
+|-|-
+|"prebuilt-layout"| Text extraction, selection marks, tables
+|"prebuilt-document"| Text extraction, selection marks, tables, key-value pairs and entities
+|"prebuilt-invoices"| Text extraction, selection marks, tables, and prebuilt fields and values pertaining to English invoices
+|"prebuilt-businessCard"| Text extraction and prebuilt fields and values pertaining to English business cards
+|"prebuilt-idDocument"| Text extraction and prebuilt fields and values pertaining to US driver licenses and international passports
+|"prebuilt-receipt"| Text extraction and prebuilt fields and values pertaining to English sales receipts
+|"{custom-model-id}"| Text extraction, selection marks, tables, labeled fields and values from your custom documents
+
+Sample code snippets to illustrate using a DocumentAnalysisClient [here](#analyze-documents-using-a-custom-model "Analyze Documents Using a Custom Model").
 
 ### DocumentModelAdministrationClient
 The [DocumentModelAdministrationClient][document_model_admin_sync_client] and
 [DocumentModelAdministrationAsyncClient][document_model_admin_sync_client] provide both synchronous and asynchronous operations
 - Build custom document analysis models to analyze text content, fields and values found in your custom documents. See example [Build a model](#build-a-model).
- A `DocumentModel` is returned indicating the document types that the model can analyze, and the fields and schemas it will extract in
-each doc type.
-- Managing models created in your account by building, listing, deleting, and see how close you are to the limit of 
-custom models your account can hold. See example [Manage models](#manage-your-models).
+  A `DocumentModel` is returned indicating the document types that the model can analyze, and the fields and schemas it will extract in
+  each doc type.
+- Managing models created in your account by building, listing, deleting, and see how close you are to the limit of
+  custom models your account can hold. See example [Manage models](#manage-your-models).
 - Copying a custom model from one Form Recognizer resource to another.
 - Creating a composed model from a collection of existing built models.
-- Listing all document model operations associated with the Form Recognizer resource.
+- Listing document model operations associated with the Form Recognizer resource.
 
-### Long-Running Operations
+Please note that models can also be trained using a graphical user interface such as the [Form Recognizer Labeling Tool][fr-labeling-tool].
```

Comment:
looks like `fr-labeling-tool` doesn't exist. link is https://aka.ms/azsdk/formrecognizer/labelingtool

Reply:
It does exist but it's named fr_labeling_tool. I will correct this.